### DISCUSSION TITLE
Block quotes with code blocks

### DISCUSSION THREAD

**Discussion by PadowYT2** [requarks/wiki#5381] [Error / Bug Report] [answered]
When trying to make a color for the block quote which has a code block doesn't work. How can I make it?
```
> Some text
> ```html
> <a></a>
> ```
{.is-danger}
```

**Comment by RED-CAD-Solutions-AG-Development** [comment]
I reported this → https://github.com/requarks/wiki/discussions/6094

**Reply by PadowYT2** [reply]
Why duplicate the issue?

**Reply by PadowYT2** [reply]
Ah well it is different mb

**Comment by NGPixel** [comment, marked as answer]
https://github.com/requarks/wiki/discussions/4732#discussioncomment-4783499

**Reply by PadowYT2** [reply]
Unfortunatly that didn't help me
```
> You will need a database setup and a user with the correct permissions created for that database before continuing any further. See below to create a user and database for your website quickly. To find more detailed information please have a look at [Setting up MySQL](/mysql).
> ```sql
> mysql -u root -p
> # Remember to change 'yourPassword' below to be a unique > password
> CREATE USER 'nameless'@'127.0.0.1' IDENTIFIED BY 'yourPassword';
> CREATE DATABASE nameless;
> GRANT ALL PRIVILEGES ON nameless.* TO 'nameless'@'127.0.0.1' WITH GRANT OPTION;
> exit
> ```
<!-- {blockquote:.is-info} -->
```

**Reply by PadowYT2** [reply]
nor did your answer

**Reply by NGPixel** [reply]
Are you running the latest version?

**Reply by PadowYT2** [reply]
@derkades

**Reply by Derkades** [reply]
well then I will do @samerton :)

**Reply by samerton** [reply]
We were on 2.5.286, now on 2.5.296 (latest) - try now @PadowYT2 to see if that's helped at all?

**Reply by PadowYT2** [reply]
It worked, but it looks ugly
![image](https://user-images.githubusercontent.com/71085027/217460875-33d05d7c-ffdf-4e21-adbc-1c6a12484522.png)

**Reply by NGPixel** [reply]
There's a PR with a fix for it that should land in the next update.
